makeflow/combined_workflow.jx

```diff
@@ -0,0 +1,250 @@
+        "METADATA": METADATA_CLEANED,
+        "WORKING_SPACE": TIFS_DIR
+    },
+      "inputs": [
+        TIFS_DIR,
+        RIGHT_BIN,
+        METADATA_CLEANED
+      ],
+      "outputs": [
+        RIGHT_TIF
+      ]
+    },
+    {
+      # Make directory to store SOILMASK files
+      "command": "mkdir -p ${SOILMASK_DIR}",
+      "environment": {
+        "SOILMASK_DIR": SOILMASK_DIR
+      },
+      "inputs": [],
+      "outputs": [
+        SOILMASK_DIR
+      ]
+    },
+    {
+      # Generate soil mask from LEFT TIFF image
+      "command": "docker run --rm -v $(pwd):/mnt -w /mnt agpipeline/soilmask:latest --result print --metadata ${METADATA} --working_space ${WORKING_SPACE} ${LEFT_TIF}",
+      "environment": {
+        "LEFT_TIF": LEFT_TIF,
+        "METADATA": METADATA_CLEANED,
+        "WORKING_SPACE": SOILMASK_DIR
+    },
+      "inputs": [
+        SOILMASK_DIR,
+        LEFT_TIF,
+        METADATA_CLEANED
+      ],
+      "outputs": [
+        LEFT_SOILMASK
+      ]
+    },
+    {
+      # Generate soil mask from RIGHT TIFF image
+      "command": "docker run --rm -v $(pwd):/mnt -w /mnt agpipeline/soilmask:latest --result print --metadata ${METADATA} --working_space ${WORKING_SPACE} ${RIGHT_TIF}",
+      "environment": {
+        "RIGHT_TIF": RIGHT_TIF,
+        "METADATA": METADATA_CLEANED,
+        "WORKING_SPACE": SOILMASK_DIR
+    },
+      "inputs": [
+        SOILMASK_DIR,
+        RIGHT_TIF,
+        METADATA_CLEANED
+      ],
+      "outputs": [
+        RIGHT_SOILMASK
+      ]
+    },
+    {
+      # Make directory to store FIELDMOSAIC files
+      "command": "mkdir -p ${FIELDMOSAIC_DIR}",
+      "environment": {
+        "FIELDMOSAIC_DIR": FIELDMOSAIC_DIR
+      },
+      "inputs": [],
+      "outputs": [
+        FIELDMOSAIC_DIR
+      ]
+    },
+    {
+      # Create MOSAIC_LIST_FILE file from the soilmask files
+      "command": "ls ${SOILMASK_DIR}*.tif > ${MOSAIC_LIST_FILE}",
+      "environment": {
+        "SOILMASK_DIR": SOILMASK_DIR,
+        "MOSAIC_LIST_FILE": MOSAIC_LIST_FILE
+      },
+      "inputs": [
+        SOILMASK_DIR,
+        FIELDMOSAIC_DIR,
+        LEFT_SOILMASK,
+        RIGHT_SOILMASK
+      ],
+      "outputs": [
+        MOSAIC_LIST_FILE
+      ]
+    },
+    {
+      # Generate field mosaic from soil mask TIFFs
+      "command": "docker run --rm -v $(pwd):/mnt -w /mnt agpipeline/fieldmosaic:latest --result print --metadata ${METADATA} --working_space ${WORKING_SPACE} ${MOSAIC_LIST_FILE} ${SENSOR} \"${MOSAIC_BOUNDS}\"",
+      "environment": {
+        "MOSAIC_LIST_FILE": MOSAIC_LIST_FILE,
+        "METADATA": METADATA_CLEANED,
+        "WORKING_SPACE": FIELDMOSAIC_DIR,
+        "SENSOR": SENSOR,
+        "MOSAIC_BOUNDS": MOSAIC_BOUNDS
+    },
+      "inputs": [
+        FIELDMOSAIC_DIR,
+        MOSAIC_LIST_FILE,
+        METADATA_CLEANED
+      ],
+      "outputs": [
+        FIELDMOSAIC_DIR + "fullfield_mosaic.png",
+        FIELDMOSAIC_DIR + "fullfield_mosaic.png.aux.xml",
```

Comment:
*.png.aux.xml is not a needed result - it's a result of the transformation app.

Reply:
Should we delete it then?

Reply:
I think it can be safely deleted when the makeflow cleanup happens